app/src/main/java/campus/tech/kakao/map/MainActivity.kt

```diff
@@ -1,58 +1,195 @@
-            startActivity(intent)
-        }
+    private fun moveCamera(latLng: LatLng) {
+        val cameraUpdate = CameraUpdateFactory.newCenterPosition(latLng)
+        Log.d("MainAct State", "Intent is: $intent")
+        Log.d("kakaomap", "moveCamera: $kakaoMap")
+        kakaoMap?.moveCamera(cameraUpdate, CameraAnimation.from(500, true, true))
+    }
+
+    private fun makeLabelStyle() {
+        vectorToBitmap(R.drawable.pin_drop_24px)
+
+        styles = LabelStyles.from(
+            "myLabel",
+            LabelStyle.from(marker)
+                .setTextStyles(32, Color.BLACK, 1, Color.GRAY)
+                .setZoomLevel(kakaoMap!!.minZoomLevel)
+        )
+        styles = kakaoMap.labelManager?.addLabelStyles(styles)!!
+    }
+
+    private fun vectorToBitmap(drawableId: Int) {
+        val drawable = ContextCompat.getDrawable(this, drawableId)
+        val bitmap = Bitmap.createBitmap(
+            drawable!!.intrinsicWidth,
+            drawable!!.intrinsicHeight,
+            Bitmap.Config.ARGB_8888
+        )
+        val canvas = Canvas(bitmap)
+        drawable.setBounds(0, 0, canvas.width, canvas.height)
+        drawable.draw(canvas)
+        marker = bitmap
+    }
+
+    private fun showBottomSheet(place: Place) {
+        bottomSheetBehavior.state = BottomSheetBehavior.STATE_COLLAPSED
+        bottomSheetName.text = place.name
+        bottomSheetAddress.text = place.address
+        bottomSheetCategory.text = place.category
+    }
+
+    private fun initBottomSheet() {
+        bottomSheetBehavior = BottomSheetBehavior.from(bottomSheetLayout)
+        bottomSheetBehavior.addBottomSheetCallback(object :
+            BottomSheetBehavior.BottomSheetCallback() {
+            override fun onStateChanged(bottomSheet: View, newState: Int) {
+                binding.searchWindow.isInvisible = (newState < BottomSheetBehavior.STATE_COLLAPSED)
+                when (newState) {
+                    BottomSheetBehavior.STATE_HIDDEN -> {
+                        bottomSheetBehavior.state = BottomSheetBehavior.STATE_COLLAPSED
+                    }
+                }
+            }
+
+            override fun onSlide(p0: View, p1: Float) {
+            }
+        })
+        bottomSheetBehavior.state = BottomSheetBehavior.STATE_HIDDEN
     }
 
     override fun onResume() {
         super.onResume()
         binding.mapView.resume()
+        Log.d("MainAct State", "Intent is: $intent")
+
     }
 
     override fun onPause() {
         super.onPause()
         binding.mapView.pause()
     }
+
+    override fun onNewIntent(intent: Intent) {
+        super.onNewIntent(intent)
+        intent?.let {
+            var place: Place?
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+                place = intent.getParcelableExtra("place", Place::class.java)
+            } else {
+                place = intent.getParcelableExtra("place") as Place?
+            }
```

Comment:
안드로이드에서 제공하는 IntentCompat 클래스에 SDK 분기처리가 되어있는 함수를 이용해보셔도 좋겠네요!
```suggestion
     intent = IntentCompat.getParcelableExtra(it, "place", Place::class.java)
```